packages/network/libmnl/package.mk

```diff
@@ -9,5 +9,6 @@ PKG_SITE="http://netfilter.org/projects/libmnl"
 PKG_URL="http://netfilter.org/projects/libmnl/files/$PKG_NAME-$PKG_VERSION.tar.bz2"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="A minimalistic user-space library oriented to Netlink developers."
+PKG_BUILD_FLAGS="+pic"
 
-PKG_CONFIGURE_OPTS_TARGET="--disable-shared --enable-static"
+PKG_CONFIGURE_OPTS_TARGET="--enable-shared --disable-static"
```

Comment:
`--disable-static --enable-shared` is the default (set by `scripts/build`) so you could drop this last line entirely to get the behaviour you want...

Reply:
tested removal (along with the +fpic) and all is well, so amended.